actionpack/lib/action_controller/metal/renderers.rb

```diff
@@ -25,9 +42,16 @@ module Renderers
     # Default values are <tt>:json</tt>, <tt>:js</tt>, <tt>:xml</tt>.
     RENDERERS = Set.new
```

Comment:
Thanks. I've decided I don't like the impl with the renderer and serializer
being one-to-one, but haven't settled on anything better, so those ideas
are still just in comments. I feel like I still haven't clarified what's
missing from the example impl. I think part of it is the role responders
played. The more minds the better!
On Wed, Feb 17, 2016 at 9:34 AM David Heinemeier Hansson <
notifications@github.com> wrote:

> In actionpack/lib/action_controller/metal/renderers.rb
> <https://github.com/rails/rails/pull/21496#discussion_r53180311>:
>
> > @@ -25,9 +42,16 @@ module Renderers
> >      # Default values are <tt>:json</tt>, <tt>:js</tt>, <tt>:xml</tt>.
> >      RENDERERS = Set.new
>
> Let me give this some thought. Definitely want to make some amendments
> here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails/rails/pull/21496/files#r53180311>.
>